docs/site/Sequence.md

```diff
@@ -229,7 +229,7 @@ How to get query string param values.
 
 ### Parsing Requests
```

Comment:
If we're going to have a subpage for this section, we should get rid of its mention entirely here.
Maybe we can also consider adding a link

Reply:
important catch! thx